### PR TITLE
first pass at adding shielded nodes, secure boot, binary auth, db encryption

### PR DESCRIPTION
**Pull request by smark88**

### CODE REVIEW

**Review by fboulnois — APPROVED** [review]
lgtm

File: terraform-modules/k8s-node-pool/variables.tf [hunk 1]
```diff
@@ -90,3 +90,13 @@ variable upgrade_settings {
...
 }
+
+variable enable_integrity_monitoring {
+  type    = bool
+  default = true
+}
+
+variable enable_secure_boot {
+  type    = bool
+  default = false
```
Comment (nmalfroy) [inline terraform-modules/k8s-node-pool/variables.tf:101]: We don't want this to be true by default?

**Review by nmalfroy — APPROVED** [review]
lgtm!

File: terraform-modules/k8s-node-pool/variables.tf (hunk 1 above)
Comment (smark88) [reply terraform-modules/k8s-node-pool/variables.tf:101]: default is off as others will use this module

**Pull request merged**